files/sshd_config

```diff
@@ -10,24 +10,25 @@
 # possible, but leave them commented.  Uncommented options override the
 # default value.
 
+% if node.os != node.OS_FAMILY_DEBIAN or node.os_version > 10:
 Include /etc/ssh/sshd_config.d/*.conf
-
-% if node.metadata.get('openssh', {}).get('port', False):
-Port ${node.metadata['openssh']['port']}
-% else:
-#Port 22
 % endif
+
+Port ${node.metadata.get('openssh').get('port')}
 #AddressFamily any
 #ListenAddress 0.0.0.0
 #ListenAddress ::
 
-#HostKey /etc/ssh/ssh_host_rsa_key
-#HostKey /etc/ssh/ssh_host_ecdsa_key
-#HostKey /etc/ssh/ssh_host_ed25519_key
+% if node.metadata.get('openssh').get('sign_host_keys').get('enabled'):
+% for host_key in node.metadata.get('openssh').get('sign_host_keys').get('keys'):
+HostKey ${host_key}
+HostCertificate ${host_key}.pub.crt
+% endfor
+% endif
 
 # Ciphers and keying
 #RekeyLimit default none
-% if node.metadata.get('openssh', {}).get('only_allow_secure_ciphers', True):
+% if node.metadata.get('openssh').get('only_allow_secure_ciphers'):
```

Comment:
if the only_allow_secure_ciphers is not set, this will throw an error and not return false.